extension/lib/shopify.api.js

```diff
@@ -0,0 +1,52 @@
+/**
+ * @param {object} config
+ * @return {{}}
```

Comment:
?

Reply:
Auto-Generated from IDE, change it to @returns {function}